notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb

```diff
@@ -1077,15 +1029,15 @@
         "\n",
         "Based on the integrated gradients method, XRAI assesses overlapping regions of the image to create a saliency map, which highlights relevant regions of the image rather than pixels.\n",
         "\n",
-        "Use Cases:\n",
+        "Use cases:\n",
         "\n",
         "  - Classification on image data.\n",
         "\n",
         "Parameters:\n",
         "\n",
         "- `step_count`: This is the number of steps to approximate the remaining sum. The more steps, the more accurate the integral approximation. The general rule of thumb is 50 steps, but as you increase so does the compute time.\n",
         "\n",
-        "In the next code cell, set the variable `XAI` to which explainabilty algorithm you will use on your custom model."
+        "In the next code cell, set the variable `XAI` to which explainabilty algorithm you use on your custom model."
```

Comment:
Change "which" to "the"

- In the next code cell, set the variable `XAI` to the explainabilty algorithm you use on your custom model.